content/desktop/release-notes.md

```diff
@@ -21,6 +21,51 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1) to see what's coming next.
 
+## 4.32.0
+
+{{< release-date date="2024-06-27" >}}
+
+{{< desktop-install all=true beta_win_arm=true version="4.32.0" build_path="/TBD/" >}}
+
+### New
+
```

Comment:
Missing:
Beta: Volumes Scheduled Backups & Cloud Export
Beta: Terminal in Docker Desktop